src/main/webapp/WEB-INF/views/drivers/add.jsp

```diff
@@ -10,10 +10,13 @@
 <body>
 <form method="post" id="driver" action="${pageContext.request.contextPath}/drivers/add"></form>
 <h1 class="table_dark">Add driver:</h1>
+<%@include file="../header.jsp"%>
```

Comment:
I guess this page should have logout option because it alternative for register page, its a bad idea showing logout when you haven't register yet